### PR TITLE
Mr serverless ga

### PR DESCRIPTION
**Pull request by gemma-shay**
https://cockroachlabs.atlassian.net/browse/DOC-7040
https://cockroachlabs.atlassian.net/browse/DOC-6774
Update MR serverless docs from preview to GA
Account for cross-region networking charges

### CODE REVIEW

File: src/current/cockroachcloud/connect-to-a-serverless-cluster.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ AWS PrivateLink for CockroachDB {{ site.data.products.serverless }} is in **[lim
 
 Amazon Web Services (AWS) PrivateLink support allows customers to establish SQL access to their clusters entirely through private AWS infrastructure, without exposure to the public internet, affording enhanced security and performance.
 
-AWS PrivateLink is available only for multiregion CockroachDB {{ site.data.products.serverless }} clusters deployed on AWS.
+AWS PrivateLink is available only for multi-region CockroachDB {{ site.data.products.serverless }} clusters deployed on AWS.
```
Comment (marksoper) [inline src/current/cockroachcloud/connect-to-a-serverless-cluster.md:37]: @gemma-shay PrivateLink is still Limited Access until Feb - shouldn't this be a "contact sales/support if you want PrivateLink for Serverless"?

**Review by marksoper — APPROVED** [review]
Approved with 1 suggestion.

File: src/current/cockroachcloud/plan-your-cluster-serverless.md [hunk 2]
```diff
@@ -66,7 +62,12 @@ Databases created in CockroachDB {{ site.data.products.serverless }} will automa
 
 Storage for a multi-region cluster is billed at the same rate as a single-region cluster. However, by default data is replicated three times in the primary region and once in each additional region, and each replica in the additional regions will accrue more storage costs. For example, a three-region cluster with data replicated five times will use 5/3 times the storage space of a single-region cluster where data is replicated three times.
 
+Cross-region operations will use additional RUs because of the cost of cross-region networking. Cross-region networking costs depend on the source and destination regions. For read operations, the source region contains the replica and the destination region is the gateway region. For write operations, the source region is the gateway region and the destination region contains the replica. There is a network charge for each replica to which an operation writes. Refer to [Pricing](https://www.cockroachlabs.com/pricing) for a matrix of cross-region costs.
```
Comment (mdlinville) [inline src/current/cockroachcloud/plan-your-cluster-serverless.md:65]:
```suggestion
Cross-region operations consume additional RUs for cross-region networking. Cross-region networking costs depend on the source and destination regions.

- For read operations, the source region contains the replica and the destination region is the gateway region.
- For write operations, the source region is the gateway region and the destination region contains the replica. There is a network charge for each replica to which an operation writes.

Refer to [Pricing](https://www.cockroachlabs.com/pricing) for a matrix of cross-region costs.
```

File: src/current/cockroachcloud/plan-your-cluster-serverless.md [hunk 3]
```diff
@@ -66,7 +62,12 @@ Databases created in CockroachDB {{ site.data.products.serverless }} will automa
...
+
+Keep in mind the following key points when planning your multi-region CockroachDB {{ site.data.products.serverless }} application's architecture:
+
 - Write-heavy applications may experience a significant increase in RU consumption because replicating writes across all regions consumes more resources.
 - Read-heavy applications may experience a smaller increase in RU consumption because the resources required to read from a single region of a multi-region cluster are comparable with a single-region cluster.
+- Cross-region reads are an anti-pattern and may significantly increase RU consumption. Features such as [global tables](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/global-tables.md), [regional by row tables](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/regional-tables.md), and [follower reads](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/follower-reads.md) make it possible to avoid most cross-region reads.
```
Comment (mdlinville) [inline src/current/cockroachcloud/plan-your-cluster-serverless.md:71]:
```suggestion
- Cross-region reads are an anti-pattern and may significantly increase RU consumption. Features such as [global tables](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/global-tables.md), [regional by row tables](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/regional-tables.md), and [follower reads](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/follower-reads.md) help avoid most cross-region reads.
```

File: src/current/cockroachcloud/plan-your-cluster-serverless.md [hunk 4]
```diff
@@ -66,7 +62,12 @@ Databases created in CockroachDB {{ site.data.products.serverless }} will automa
...
+- Cross-region writes will also consume additional RUs, but should not significantly increase consumption.
```
Comment (mdlinville) [inline src/current/cockroachcloud/plan-your-cluster-serverless.md:72]: This feels like duplication from ~ line 65.

File: src/current/cockroachcloud/serverless-resource-usage.md [hunk 5]
```diff
@@ -114,6 +114,7 @@ The following recommendations can help reduce the RU cost of a query by reducing
 - Use range `UPDATE` and `DELETE` statements to affect many rows in a single statement, rather than sending a separate statement per row.
 - Avoid returning columns that your application does not need.
 - Don't disable automatic statistics, as they are needed to power the [optimizer](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/cost-based-optimizer).
+- For multi-region clusters, avoid cross-region reads by using features such as [global tables](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/global-tables.md), [regional by row tables](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/regional-tables.md), and [follower reads](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/follower-reads.md) where possible.
```
Comment (mdlinville) [inline src/current/cockroachcloud/serverless-resource-usage.md:117]: Is it worth linking to the section in plan-your-serverless-cluster.md for more details?

**Review by mdlinville — APPROVED** [review]
LGTM, a few optional suggestions.

**Pull request merged**